models/shreds.py

```diff
@@ -11,19 +11,14 @@
 
 
 class Features(EmbeddedDocument):
-    on_sheet_x = IntField(default=0)
```

Comment:
I might be wrong here, but this wouldn't remove it, until you remove it from Shred object that is produced by unshred:
https://github.com/dchaplinsky/unshred-tag/blob/master/cli/load_to_mongo.py#L146